lib/api/src/modules/refs.ts

```diff
@@ -46,7 +46,7 @@ export interface ComposedRef {
 }
 export interface ComposedRefUpdate {
   title?: string;
-  type?: 'auto-inject' | 'unknown' | 'lazy';
+  type?: 'auto-inject' | 'unknown' | 'lazy' | 'server-checked';
```

Comment:
Should this at least be an enum we reuse?

Reply:
I'll refactor so it's re-used. All sub-types are re-usable even if not individually exported.